SwiftificationTests/StringTests.swift

```diff
@@ -87,4 +87,21 @@ class StringTests: XCTestCase {
         XCTAssertNil(str[safe: str.startIndex.advancedBy(5, limit: str.endIndex)])
     }
     
-}
\ No newline at end of file
+    func testAppendAndPrependOptionalString() {
+        let str1: String? = "Hi"
+        let str2: String? = " Bob "
+        let strNil: String? = nil
+        
+        let prependString1On2 = str1 + str2 // "Hi Bob "
```

Comment:
Naming these doesn't really buy you anything because you can just do `XCTAssertEqual(str1 + str2, "Hi Bob ")`, which is both shorter and (IMO) clearer.